docs/code/EntryPoints.md

```diff
@@ -0,0 +1,188 @@
+﻿# Overview
+
+An 'entry point', is a representation of a ML.Net type in json format and it is used to serialize and deserialize an ML.Net type in JSON. 
```

Comment:
>it [](start = 74, length = 2)

Ambiguous, when we say "it" what are we referring to? Entry-points? JSON? An ML.NET type?